scripts/scripts.js

```diff
@@ -751,6 +751,13 @@ export async function fetchBlogArticleIndex() {
   return (index);
 }
 
+export function makeLinkRelative(href) {
+  const url = new URL(href);
+  const host = url.hostname;
+  if (host.endsWith('.page') || host.endsWith('.live') || host === 'business.adobe.com') return (url.pathname);
```

Comment:
Should we also do localhost?

Also, what happens if I link to `https://main--blog--adobe.hlx3.page/announcing-project-helix` won't this be converted to relative?

Reply:
I would probably do:
```js
if (host.endsWith('.page') || host.endsWith('.live') || host === 'business.adobe.com') return (`${url.pathname}${url.search}`);
```